src/metabase/models/interface.clj

```diff
@@ -251,6 +251,28 @@
   {:in  json-in
    :out json-out-without-keywordization})
 
+(defn transform-validator
+  "Given a transform, returns a transform that call `assert-fn` on the value.
```

Comment:
```suggestion
  "Given a transform, returns a transform that call `assert-fn` on the "out" value.
```

Not sure how better to refer to which side we validate on, but this helper is too ambiguous without qualifying it some way. 

I thought about perhaps "transformed value", but  the designation of `in` and `out` in our transforms feel so flipped to me (e.g. "out" being what we do to values coming *into* the application). It feels like I'm missing some analogy or other use case 🤔